static/src/javascripts/projects/commercial/modules/close-disabled-slots.js

```diff
@@ -1,9 +1,25 @@
 // @flow
 import qwery from 'qwery';
 import fastdom from 'lib/fastdom-promise';
+import once from 'lodash/functions/once';
+import { dfpEnv } from 'commercial/modules/dfp/dfp-env';
 import { commercialFeatures } from 'commercial/modules/commercial-features';
 
-const adSlotSelector: string = '.js-ad-slot';
+const shouldDisableAdSlot = adSlot =>
+    window.getComputedStyle(adSlot).display === 'none';
+
+const closeDisabledSlots = once((): Promise<void> => {
+    // Get all ad slots
+    let adSlots: Array<Element> = qwery(dfpEnv.adSlotSelector);
```

Comment:
I lifted this code from another file, but yes. Another pr with fastdoms is worth doing.